data/json/monsters/zed_misc.json

```
@@ -1,4 +1,85 @@
 [
+  {
+    "id": "mon_frog_mother",
+    "type": "MONSTER",
+    "name": { "str": "frog mother" },
+    "description": "A large, bulbous creature with an extremely large mouth and small, stubby legs with no feet.  It is covered in an oily greenish-black residue and large, white pustules that seem ready to pop.",
+    "default_faction": "zombie",
+    "species": [ "ZOMBIE", "AMPHIBIAN" ],
+    "volume": "240 L",
+    "weight": "300 kg",
+    "hp": 600,
+    "speed": 75,
+    "material": [ "flesh" ],
+    "symbol": "F",
+    "color": "black_green",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 1,
+    "melee_dice": 1,
```

```suggestion
    "melee_dice": 5,
```
Neither the momma nor the tadpoles are terribly threatening right now, and won't be hanging in groups with other zombies. I would suggest making the momma able to cause some real harm, and having the tadpoles buff her so that you're trying to deal with them while she gnaws on you.